src/coreclr/jit/gentree.h

```diff
@@ -4927,6 +4927,9 @@ struct GenTreeJitIntrinsic : public GenTreeOp
     GenTreeJitIntrinsic(
         genTreeOps oper, var_types type, GenTree* op1, GenTree* op2, CorInfoType simdBaseJitType, unsigned simdSize)
         : GenTreeOp(oper, type, op1, op2)
+        , gtLayout(nullptr)
+        , gtAuxiliaryJitType(CORINFO_TYPE_UNDEF)
```

Comment:
This is needed due to an assert encountered while running PMI diffs, its related to the refactoring that happened in https://github.com/dotnet/runtime/pull/50832.
If these aren't explicitly zeroed, they may be `0xDD` in debug builds and cause assertions when calling the helper functions used in hashing the trees.